index.bs

```diff
@@ -769,12 +773,28 @@ console.log(&#96;values: ${bufferE}&#96;);
 </pre>
 </div>
 
+### The {{MLNamedArrayBufferViews}} transfer algorithm ### {#mlnamedarraybufferviews-transfer}
+To <dfn for="MLNamedArrayBufferViews">transfer</dfn> an {{MLNamedArrayBufferViews}} |views|:
+1. Let |transferredViews| be a new {{MLNamedArrayBufferViews}}.
+1. For each |key| -> |value| of |views|:
+    1. Let |transferredBuffer| be the result of [=transfer array buffer|transferring=] {{ArrayBuffer}} |value|.\[[ViewedArrayBuffer]].
+    1. Let |constructor| be the appropriate [=view constructor=] for the type of {{ArrayBufferView}} |value|.
+    1. Let |elementsNumber| be the result of |value|.\[[ByteLength]] ÷ [=element size=] of |value|.
```

Comment:
Use https://webidl.spec.whatwg.org/#buffersource-byte-length instead of accessing [[ByteLength]] directly.

Reply:
Thanks for the pointer. Fixed it in the latest commit.